### PR TITLE
Bug 1805946 - Ensure Glean and Nimbus gradle plugins are consuming the latest version

### PR DESCRIPTION
**Pull request by gabrielluong**
### Pull Request checklist

- [ ] **Quality**: This PR builds and passes detekt/ktlint checks (A pre-push hook is recommended)
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Changelog**: This PR includes [a changelog entry](https://github.com/mozilla-mobile/android-components/blob/main/docs/changelog.md) or does not need one
- [ ] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/main/android/accessibility_guide.md) or does not include any user facing features

### After merge
- **Milestone**: Make sure issues closed by this pull request are added to the [milestone](https://github.com/mozilla-mobile/android-components/milestones) of the version currently in development.
- **Breaking Changes**: If this is a breaking change, please push a draft PR on [Reference Browser](https://github.com/mozilla-mobile/reference-browser) to address the breaking issues.

### GitHub Automation

Used by GitHub Actions.

### CODE REVIEW

File: android-components/plugins/publicsuffixlist/settings.gradle [hunk 1]
```diff
@@ -2,6 +2,4 @@
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
-object AndroidComponents {
-    const val VERSION = "109.0.20221211143351"
-}
+// Prevents gradle builds from looking for a root settings.gradle
```
Comment (csadilek) [inline android-components/plugins/publicsuffixlist/settings.gradle:5]: Weird that this looks like the file moved. 🤔

**Review by csadilek — APPROVED** [review]

File: android-components/plugins/publicsuffixlist/settings.gradle (hunk 1 above)
Comment (csadilek) [reply android-components/plugins/publicsuffixlist/settings.gradle:5]: (In the diff view.)
Screenshot 2022-12-15 at 12 28 18 PM

Comment (gabrielluong) [reply android-components/plugins/publicsuffixlist/settings.gradle:5]: I know, but it should be fine.

Comment (csadilek) [reply android-components/plugins/publicsuffixlist/settings.gradle:5]: Yeah, let's land it.

**Pull request merged**